transformation_test/testdata/logging_processor-hadoop/input.log

```diff
@@ -0,0 +1,4 @@
+2022-02-01 18:09:47,136 INFO org.apache.hadoop.hdfs.server.namenode.FSEditLog: Edit logging is async:true
+2022-02-01 18:09:48,200 WARN org.apache.hadoop.hdfs.server.datanode.DataNode: Warning message here
+2022-02-01 18:09:49,300 ERROR org.apache.hadoop.hdfs.server.namenode.FSNamesystem: Error occurred
+2022-02-01 18:09:50,400 DEBUG org.apache.hadoop.hdfs.server.datanode.DataNode: Debug message
```

Comment:
Please add a multiline log entry so we can test the "multiline parsing" functionality of this processor.